astropy/_erfa/erfa_additions.h

```diff
@@ -0,0 +1,8 @@
+#ifndef ERFAADDITIONSDEF
+#define ERFAADDITIONSDEF
+
+/* Extra/PVMergeExtract */
+void eraPav2pv(double p[3], double v[3], double pv[2][3]);
+void eraPv2pav(double pv[2][3], double p[3], double v[3]);
```

Comment:
Could you add a comment to this file about why it is needed?

Reply:
Done.